charts/csi-vxflexos/values.yaml

```diff
@@ -30,13 +30,13 @@ images:
     image: registry.k8s.io/sig-storage/csi-external-health-monitor-controller:v0.13.0
   # CSM sidecars
   replication:
-    image: quay.io/dell/container-storage-modules/dell-csi-replicator:v1.10.0
+    image: quay.io/dell/container-storage-modules/dell-csi-replicator:v1.11.0
   vgsnapshotter:
-    image: quay.io/dell/container-storage-modules/csi-volumegroup-snapshotter:v1.7.0
+    image: quay.io/dell/container-storage-modules/csi-volumegroup-snapshotter:v1.8.0
   podmon:
-    image: quay.io/dell/container-storage-modules/podmon:v1.11.0
+    image: quay.io/dell/container-storage-modules/podmon:v1.12.0
   authorization:
-    image: quay.io/dell/container-storage-modules/csm-authorization-sidecar:v1.12.0
+    image: quay.io/dell/container-storage-modules/csm-authorization-sidecar:v1.13.0
```

Comment:
Authorization sidecar images can move to v2.1.0

Reply:
Done